src/sections/referrals/components/CodeInput/CodeInput.tsx

```diff
@@ -44,7 +49,16 @@ export const CodeInput = forwardRef<HTMLInputElement, InputProps>(
             {t("referrals.button.randomCode")}
           </Button>
         )}
+        <Spacer size={2} />
         {error && <SErrorMessage>{error}</SErrorMessage>}
+        {!error && registrationFee.data && (
+          <Text fs={12} color="brightBlue300">
+            {t("referrals.button.linkFee", {
+              amount: registrationFee.data?.amount,
+              symbol: registrationFee.data.symbol,
+            })}
+          </Text>
+        )}
```

Comment:
When I start typing, the error appears and the input field jumps. It's because error message is absolute positioned, but the "cost" message is not.

Can you fix that?